packages/client/lib/rpc/modules/eth.ts

```diff
@@ -423,7 +437,7 @@ export class Eth {
     if (latest < blockNumberBN) {
       return {
         code: INVALID_PARAMS,
-        message: `specified block greater than current height`,
+        message: `Specified block greater than current height`,
```

Comment:
why capitalize? the other error messages seem to start lowercase

Reply:
Ah actually I thought the others were capitalized, e.g. all the: "message: `Currently only "latest" block supported`", it seems like it's not consistent though so yeah no need to capitalize.

Reply:
oops yeah we were looking at different ones, for new messages it's okay but in general it wouldn't be good practice to change existing messages since it would disrupt anyone possibly doing exact string matches (not that they should, but better safe than sorry)